docs/helm_apis.md

```diff
@@ -0,0 +1,108 @@
+**List Helm Releases**
```

Comment:
I'm good with a `/docs` dir, might suggest you make it a `/docs/helm` dir, and make this your main `README.md` file inside.

Reply:
Done